packages/react/src/global-alert/GlobalAlertCloseButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { MouseEventHandler } from 'react';
+import { Button } from '../button';
+import { mapSpacing, mq, tokens } from '../core';
+import { CloseIcon } from '../icon';
+
+type GlobalAlertCloseButtonProps = {
+	onClick?: MouseEventHandler<HTMLButtonElement>;
+};
+
+export const GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_BREAKPOINT = 'lg';
+export const GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_MQ =
+	tokens.mediaQuery.min[GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_BREAKPOINT];
+
+export const GlobalAlertCloseButton = ({
+	onClick,
+}: GlobalAlertCloseButtonProps) => {
+	return (
+		<Button
+			onClick={onClick}
+			iconAfter={CloseIcon}
+			variant="text"
+			aria-label="Close"
+			css={mq({
+				position: 'absolute',
```

Comment:
Can we remove absolute positioning from here and use flexbox instead? It looks like currently the heading and close button are not aligned, but not sure if thats intentional or not.